frameworks/PHP/symfony/src/Controller/DbController.php

```diff
@@ -12,9 +12,20 @@ class DbController
 {
     /** @var EntityManagerInterface */
     private $entityManager;
+
     /** @var WorldRepository */
     private $worldRepository;
 
+    private function getUniqueRandomNumbers($count, $min, $max)
+    {
+        $res = [];
+        do {
+            $res[\mt_rand($min, $max)] = 1;
```

Comment:
remove `\` for non-optimized methods (list: https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/3048#issuecomment-328810072)

Reply:
It doesn't change anything for performance, with non-optimized functions, but using the `\` removes the ambiguity about which function to use, which is a good practice.